features/mbedtls/src/psa_crypto_core.h

```diff
@@ -0,0 +1,175 @@
+/*
+ *  PSA crypto core internal interfaces
```

Comment:
Any idea why this isn't being picked up as a move/rename by git? Was it actually not present in Mbed OS before? Or did we forget to remove the old copy? (in commit "Update Mbed TLS to latest version")

Reply:
I think I might have made a mistake when updating the Makefile. There look to be two copies of this file (and the others you've pointed this out on) now, one in `src` and one in `platform/COMPONENT_PSA_SRV_IMPL`. Is there something I missed with the Makefile changes?

Reply:
If the Makefile places this in a new location, you'll have to manually delete it using git. The Makefile won't run git to add or remove files.

Reply:
I think I've worked it out, I had combined the Makefiles badly so I was duplicating all the PSA specific headers. There shouldn't be duplicated files now.

Reply:
`crypto_compat.h` still seems to be deleting and remaking, rather than doing a move, not sure why. Maybe the file changed too much?